apps/meteor/server/lib/sendDirectMessageToUsers.ts

```diff
@@ -10,15 +10,15 @@ export async function sendDirectMessageToUsers(
 	toIds: string[],
 	messageFn: (user: IUser) => string,
 ): Promise<string[]> {
-	const fromUser = await Users.findOneById(fromId, { projection: { _id: 1 } });
+	const fromUser = await Users.findOneById(fromId, { projection: { _id: 1, username: 1 } });
 	if (!fromUser) {
 		throw new Error(`User not found: ${fromId}`);
 	}
 
 	const users = Users.findByIds(toIds, { projection: { _id: 1, username: 1, language: 1 } });
 	const success: string[] = [];
 
-	users.forEach((user: IUser) => {
+	for await (const user of users) {
```

Comment:
It looks better, but, do we need it to an async for loop? No promises are running inside 👀

Reply:
@d-gubert

Reply:
Honestly, since `await` works event if the value is not a promise, I would just keep it there because it's already there :shrug: It wasn't me who did it, btw :joy: